CONTRIBUTING.md

```diff
@@ -9,25 +28,35 @@ tox --notest -e unit
 source .tox/unit/bin/activate
 ```
 
-## Testing
-
-This project uses `tox` for managing test environments. There are some pre-configured environments
-that can be used for linting and formatting code when you're preparing contributions to the charm:
+### Testing
 
 ```shell
 tox -e fmt           # update your code according to linting rules
 tox -e lint          # code style
 tox -e unit          # unit tests
-tox -e integration   # integration tests
 tox                  # runs 'lint' and 'unit' environments
 ```
 
-## Build the charm
+## Build charm
 
 Build the charm in this git repository using:
 
 ```shell
 charmcraft pack
 ```
 
-<!-- You may want to include any contribution/style guidelines in this document>
+### Deploy
+
+
+```bash
+# Create a model
+juju add-model dev
+# Enable DEBUG logging
+juju model-config logging-config="<root>=INFO;unit=DEBUG"
+# Deploy identity-platform-login-ui charm
+juju deploy ./identity-platform-login-ui_ubuntu-*.charm --resource oci-image=$(yq eval '.resources.oci-image.upstream-source' metadata.yaml)
```

Comment:
```suggestion
juju deploy ./identity-platform-login-ui-operator_ubuntu-*.charm --resource oci-image=$(yq eval '.resources.oci-image.upstream-source' metadata.yaml)
```

Reply:
We also need to add `--trust` to the deploy command as you're using `KubernetesServicePatch` - otherwise deployment would fail when rbac is enabled

Reply:
done